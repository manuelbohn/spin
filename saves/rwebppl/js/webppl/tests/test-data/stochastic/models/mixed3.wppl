var logistic = function(x) {
  return 1 / (1 + Math.exp(-x));
};

var model = function() {
  var x = gaussian(5, 10);
  var y = flip(logistic(x));
  return y;
};
