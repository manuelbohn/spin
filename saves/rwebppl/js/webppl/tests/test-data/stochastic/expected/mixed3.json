{
  "hist": {
    "true": 0.6886,
    "false": 0.3114
  }
}
